src/mixins/editMixin.js

```diff
@@ -26,14 +26,22 @@ export default {
     hasChanged () {
       return !this.isNew && !deepEqual(this.value, this.edit)
     },
+    today () {
+      const date = new Date()
+      return date.toISOString()
+    },
   },
   methods: {
     save (event) {
       if (this.isNew) {
         this.$emit('save', this.edit, event)
       }
       else {
-        this.$emit('save', { ...objectDiff(this.value, this.edit), id: this.value.id }, event)
+        this.$emit(
+          'save',
+          { ...objectDiff(this.value, this.edit), id: this.value.id },
+          event,
+        )
```

Comment:
Could you revert this unrelated formatting change?

Reply:
Fix pushed